src/OpenTelemetry.Instrumentation.AdditionalDeps/OpenTelemetry.Instrumentation.AdditionalDeps.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Just out of curiosity, is this `OpenTelemetry.Instrumentation.AdditionalDeps.dll` also important, or it's just matter of cleanup?

Reply:
`OpenTelemetry.Instrumentation.AdditionalDeps.dll` is a part of deps.json, hence we need to add this to the store to load. There are no functionality in this library that gets invoked at runtime. This library has `DiagnosticSource` dependency and entire probing path is build based on it.

We could remove the need for `OpenTelemetry.Instrumentation.AdditionalDeps.dll`, but the deps.json need to go through a manual edit to remove the entry for `OpenTelemetry.Instrumentation.AdditionalDeps.dll`. This could be achieved by tweaking build steps. If we are inclined towards this approach, want to get this done as a part of follow up PR.

Reply:
> > Are the net5.0 references under `netcoreapp3.1/store/[Platform]/net5.0` important? ... to me it seems like it's not connected only to netcoreapp3.1, maybe worth moving to root dir?
> 
> We need the folder for all support runtimes. Currently we have netcoreapp3.1 and net5.0. When we add support for .net6.0 there will be an additional folder that gets created here. Dotnet Runtime Store functionality forces us to have folder structure for all supported runtime to load correctly.

The framework folders sounds reasonable, just if it should be inside netcoreapp3.1 dir? I was thinking more like `[root]/store/[platform]/[framework]/...`

Reply:
Just an additional idea for later. If we lift `OpenTelemetry.Instrumentation.AdditionalDeps`'s logic to `OpenTelemetry.ClrProfiler.Managed`, does this mean we can fix also all OTel.SDK packages and their dependencies automatically? ... also we don't double load different versions, in case user code uses the same packages?

Reply:
This will make both AdditionalDeps and RuntimeStore very complex. It's better to keep these separated and on a  need basis update the package that we are interested in `OpenTelemetry.Instrumentation.AdditionalDeps`.